charts/airbyte/values.yaml

```diff
@@ -332,6 +332,98 @@ server:
+  ## @param worker.livenessProbe.enabled Enable livenessProbe on the worker
+  ## @param worker.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param worker.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param worker.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param worker.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param worker.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    timeoutSeconds: 1
+    failureThreshold: 3
+    successThreshold: 1
+
+  ## @param worker.readinessProbe.enabled Enable readinessProbe on the worker
+  ## @param worker.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param worker.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param worker.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param worker.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param worker.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    timeoutSeconds: 1
+    failureThreshold: 3
+    successThreshold: 1
+
+  ## worker resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param worker.resources.limits [object] The resources limits for the worker container
+  ## @param worker.resources.requests [object] The requested resources for the worker container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 200m
+    ##    memory: 1Gi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    memory: 256Mi
+    ##    cpu: 250m
+    requests: {}
+
+  ## @param worker.nodeSelector [object] Node labels for pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## @param worker.tolerations [array] Tolerations for worker pod assignment.
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+  ## @param worker.log.level The log level to log at.
+  log:
+    level: "INFO"
+
+  ## @param worker.extraEnv [array] Additional env vars for worker pod(s).
+  ## Example:
+  ##
+  ## extraEnv:
+  ## - name: WORKER_POD_TOLERATIONS
+  ##   value: "key=airbyte-server,operator=Equals,value=true,effect=NoSchedule"
+  extraEnv: []
```

Comment:
I've been thinking more about the helm chart and how to make it extensible by end users to reduce the number of times that the helm chart would need to be republished just to allow for setting new env vars. Outside of the core variables, I think using `extraEnv` is a good solution for this, but open to suggestions. If this sounds okay, I might add this to all of the different deployments.

You can see it [here](https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml#L240-L246) in the bitnami/postgres helm chart as well.